### ISSUE TITLE
Consistent usage of keyword 'Remarks:'

### ISSUE THREAD

**Issue by hoppfrosch** [fincs/GenDocs#3]
It's possible to use `Remarks:`-Keyword within `Method`-documentation section. This way additional information can be added to `Method`-documentation. This is especially useful if you want to add structure to your remarks using Markup:

```
/*!
    Method: test(x)
        bla
    Parameters:
        x - *(Optional)* bla
    Remarks:
        ### See also: 
            [test2](#test2)

        ### Author(s)
            * 20130308 - [hoppfrosch](hoppfrosch@anywhere.com) - bla
*/
```

I failed to find a way to add something similar to the `class`\- or  `library`-documentation block ..

Wouldn't it make sense to support `Remarks:`-Keyword within `method` as well as `property`, `class` and `library` documentation (what else?) sections to get some consistency and make usage of  `Remarks:` more intuitive?

**Comment by fincs** [comment]
I agree with this, and will change it when I have time. By the way, in this particular example you should use `Extra:` instead of `Remarks:` as the latter would create an apparently empty 'Remarks' header (you're using h3 in your Remarks section, which is the same level that the Remarks section itself uses).
